src/index.js

```diff
@@ -1,7 +1,7 @@
 import path from 'path';
 
 import validateOptions from 'schema-utils';
-import log from 'webpack-log';
+import logging from 'webpack/lib/logging/runtime';
```

Comment:
Do not use this, use `compiler.getLogger()`

Reply:
you meant `compilation.getLogger()` ?
cause I guess `compiler` object has `infrastructureLogger`. Are they same ?